core/src/main/java/feign/stream/StreamDecoder.java

```diff
@@ -103,4 +114,13 @@ public Type getOwnerType() {
       return null;
     }
   }
+
+  public static class DefaultIteratorDecoder implements Decoder {
```

Comment:
What if we call it `LineToIteratorDecoder`?

Reply:
This looks pretty good.